readme.txt

```diff
@@ -1,11 +1,11 @@
-=== The Events Calendar PRO Extension: Limit Time Range in Week View ===
+=== The Events Calendar Pro Extension: Limit Week View Time Range ===
 Contributors: ModernTribe
 Donate link: http://m.tri.be/29
 Tags: events, calendar
-Requires at least: 4.5
-Tested up to: 4.9.6
-Requires PHP: 5.4
-Stable tag: 1.0.0
+Requires at least: 5.0
```

Comment:
Why does it require WP core 5.0+? If this isn't a mistake, explain in this version's changelog entry.